rdtools/filtering.py

```diff
@@ -124,3 +140,322 @@ def csi_filter(poa_global_measured, poa_global_clearsky, threshold=0.15):
 
     csi = poa_global_measured / poa_global_clearsky
     return (csi >= 1.0 - threshold) & (csi <= 1.0 + threshold)
+
+
+def format_clipping_time_series(power_ac, mounting_type):
```

Comment:
Do you think users will ever want to call `format_clipping_time_series` and `calculate_max_derivative` themselves?  If they're really just for the other functions to use, it's probably a good idea to make these ones private by prepending an underscore to the function name -- that way if we need to change them someday we won't have to cut a major release.